### PR TITLE
[docs] Advanced Federation

### PR DESCRIPTION
**Pull request by kaitlincarter-hc**
Updates:
- renamed file
- changed format, erb -> md
- Added headers
- Moved a section
- updated output (text -> sh)

### CODE REVIEW

File: website/source/docs/guides/advanced-federation.html.md [hunk 1]
```diff
@@ -7,12 +7,10 @@ description: |-
 ---
 
 # Multiple Datacenters
```
Comment (judithpatudith) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
# [Enterprise] Advanced Federation
```

File: website/source/docs/guides/advanced-federation.html.md [hunk 2]
```diff
@@ -7,12 +7,10 @@ description: |-
...
-## Advanced Federation with Network Areas
 
-[//]: # ( ~> The network area functionality described here is available only in )
-[//]: # (   [Consul Enterprise](https://www.hashicorp.com/products/consul/) version 0.8.0 and later. )
+## [Enterprise] Advanced Federation with Network Areas
```
Comment (judithpatudith) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
```

File: website/source/docs/guides/advanced-federation.html.md [hunk 3]
```diff
@@ -72,38 +61,42 @@ the WAN because they need to be available to all Consul datacenters.
 A peer datacenter can connected via the WAN gossip pool and a network area at the
 same time, and RPCs will be forwarded as long as servers are available in either.
 
-## Getting Started
+## Configure Advanced Federation
 
-To get started, follow the [bootstrapping guide](/docs/guides/bootstrapping.html) to
+To get started, follow the [Deployment guide](https://learn.hashicorp.com/consul/advanced/day-1-operations/deployment-guide) to
```
Comment (judithpatudith) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
To get started, follow the [Deployment guide](https://learn.hashicorp.com/consul/advanced/day-1-operations/deployment-guide/) to
```

File: website/source/docs/guides/advanced-federation.html.md [hunk 4]
```diff
@@ -162,3 +159,20 @@ tunneling mechanism. Consul does not handle VPN or NAT traversal for you.
...
+## Data Replication
+
+
+In general, data is not replicated between different Consul datacenters. When a
+request is made for a resource in another datacenter, the local Consul servers forward
+an RPC request to the remote Consul servers for that resource and return the results.
+If the remote datacenter is not available, then those resources will also not be
+available, but that won't otherwise affect the local datacenter. There are some special
+situations where a limited subset of data can be replicated, such as with Consul's built-in
+[ACL replication](/docs/guides/acl.html#outages-and-acl-replication) capability, or
```
Comment (judithpatudith) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
[ACL replication](/docs/guides/acl.html#outages-and-acl-replication/) capability, or
```

File: website/source/docs/guides/advanced-federation.html.md [hunk 5]
```diff
@@ -162,3 +159,20 @@ tunneling mechanism. Consul does not handle VPN or NAT traversal for you.
...
+external tools like [consul-replicate](https://github.com/hashicorp/consul-replicate).
```
Comment (judithpatudith) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
external tools like [consul-replicate](https://github.com/hashicorp/consul-replicate/).
```

File: website/source/docs/guides/advanced-federation.html.md (hunk 1 above)
Comment (kaitlincarter-hc) [reply website/source/docs/guides/advanced-federation.html.md]: Same thoughts here as the basic federation.

File: website/source/docs/guides/advanced-federation.html.md [hunk 6]
```diff
@@ -6,13 +6,10 @@ description: |-
   One of the key features of Consul is its support for multiple datacenters. The architecture of Consul is designed to promote low coupling of datacenters so that connectivity issues or failure of any datacenter does not impact the availability of Consul in other datacenters. This means each datacenter runs independently, each having a dedicated group of servers and a private LAN gossip pool.
 ---
 
-# Multiple Datacenters
-## Advanced Federation with Network Areas
+# [Enterprise] Advanced Federation
```
Comment (kaitlincarter-hc) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
# [Enterprise] Multiple Datacenter: Advanced Federation with Network Areas
```

Comment (kaitlincarter-hc) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
# [Enterprise] Multiple Datacenter: Advanced Federation with Network Areas
```

File: website/source/docs/guides/advanced-federation.html.md [hunk 7]
```diff
@@ -6,13 +6,10 @@ description: |-
   One of the key features of Consul is its support for multiple datacenters. The architecture of Consul is designed to promote low coupling of datacenters so that connectivity issues or failure of any datacenter does not impact the availability of Consul in other datacenters. This means each datacenter runs independently, each having a dedicated group of servers and a private LAN gossip pool.
 ---
 
-# Multiple Datacenters
-## Advanced Federation with Network Areas
+# [Enterprise] Multiple Datacenter: Advanced Federation with Network Areas
```
Comment (kaitlincarter-hc) [inline website/source/docs/guides/advanced-federation.html.md]:
```suggestion
# [Enterprise] Multiple Datacenters: Advanced Federation with Network Areas
```

**Pull request merged**